src/protoc-gen-php.cc

```diff
@@ -47,6 +47,7 @@
 #include <google/protobuf/io/zero_copy_stream.h>
 
 using std::string;
+using namespace std;
```

Comment:
you can get rid of the 'using std::string' above if you're importing all of std::